packages/data/src/world/mm/stone_tower_temple_inverted.yml

```diff
@@ -128,8 +128,8 @@
   dungeon: IST
   exits:
     "Stone Tower Temple Inverted Pre-Boss": "soul_enemy(SOUL_ENEMY_EYEGORE) && (has_explosives || has_arrows || can_hookshot_short || has_mask_zora || has_mask_goron)"
-    "Stone Tower Temple Inverted Boss Key Room": "(trick(MM_ISTT_EYEGORE) && (has_mask_goron || has_explosives)) || short_hook_anywhere"
-    "Stone Tower Temple Inverted Center": "(trick(MM_ISTT_EYEGORE) && (has_mask_goron || has_explosives)) || short_hook_anywhere"
+    "Stone Tower Temple Inverted Boss Key Room": "(trick(MM_ISTT_EYEGORE) && (has_mask_goron || has_explosives || has_hover_boots)) || short_hook_anywhere"
+    "Stone Tower Temple Inverted Center": "(trick(MM_ISTT_EYEGORE) && (has_mask_goron || has_explosives || has_hover_boots)) || short_hook_anywhere"
```

Comment:
we can take goron and hovers out of the trick for these.

Reply:
why?